authenticate/authenticate.cabal

```diff
@@ -19,7 +19,7 @@ flag network-uri
 
 library
     build-depends:   base                          >= 4.10     && < 5
-                   , aeson                         >= 0.5
+                   , aeson                         >= 2.0      && < 2.1
```

Comment:
I try to avoid adding lower bounds like this that will bifurcate packages. Would it be possible to update this PR to use CPP to support both old and new aeson?

Reply:
Done, `aeson` < 2.0 is now supported as well.